source/grid/manifold_lib.cc

```diff
@@ -1801,7 +1801,8 @@ namespace
         // identify the weights for the lines to be accumulated (vertex
         // weights are set outside and coincide with the flat manifold case)
 
-        std::array<double, GeometryInfo<3>::lines_per_cell> weights_lines{};
+        std::array<double, GeometryInfo<3>::lines_per_cell> weights_lines;
+        std::fill(weights_lines.begin(), weights_lines.end(), 0.0);
```

Comment:
That works. I think you could also have initialized it via `= {0.0, ...12 times...}`.

Reply:
I like the current setup.